actors/evm/tests/storage_footprint.rs

```diff
@@ -0,0 +1,242 @@
+use std::fs::File;
+use std::io::Write;
+use std::sync::Arc;
+
+use ethers::contract::Lazy;
+use ethers::prelude::abigen;
+use ethers::providers::{MockProvider, Provider};
+use fvm_ipld_blockstore::tracking::BSStats as BlockstoreStats;
+use fvm_shared::address::Address;
+
+mod env;
+
+use env::{TestContractCall, TestEnv};
+use serde_json::json;
+
+// Generate a statically typed interface for the contract.
+abigen!(StorageFootprint, "./tests/contracts/StorageFootprint.abi");
+
+// Alternatively we can generate the ABI code as follows:
+// ```
+//     ethers::prelude::Abigen::new("StorageFootprint", "./tests/contracts/StorageFootprint.abi")
+//         .unwrap()
+//         .generate()
+//         .unwrap()
+//         .write_to_file("./tests/storage_footprint_abi.rs")
+//         .unwrap();
+// ```
+
+// The owner doesn't matter in these tests, so just using the same value that the other tests use, for everything.
+const OWNER: Address = Address::new_id(100);
+
+static CONTRACT: Lazy<StorageFootprint<Provider<MockProvider>>> = Lazy::new(|| {
+    // The owner of the contract is expected to be the 160 bit hash used on Ethereum.
+    // We're not going to use it during the tests.
+    let owner_hex = format!("{:0>40}", hex::encode(OWNER.payload_bytes()));
+    let address = owner_hex.parse::<ethers::core::types::Address>().unwrap();
```

Comment:
We already have a method to convert from an "id" address to an "eth" address. Can we reuse that?

1. Make `EthAddress.0` public.
2. `ethers::core::types::Address(EthAddress::from_id_address(OWNER).unwrap().0)`.

That way, these will be _valid_ eth addresses.

Reply:
Ah great, thanks!